src/sonic-config-engine/sonic-cfggen

```diff
@@ -50,6 +50,9 @@ from redis_bcc import RedisBytecodeCache
 from collections import OrderedDict
 from natsort import natsorted
 
+PY3x = sys.version_info >= (3, 0)
+STR_TYPE = str if PY3x else unicode
```

Comment:
Suggest to add a TODO comment to remove this once we've fully transitioned to Python 3.

Reply:
ACK